geoapi/tests/conftest.py

```diff
@@ -113,7 +113,7 @@ def public_projects_fixture(projects_fixture):
 
 
 @pytest.fixture(scope="function")
-def observable_projects_fixture():
+def observable_projects_fixture(user1):
```

Comment:
```suggestion
def observable_projects_fixture():
```
oops. i didn't mean to commit this